lib/features/children/generosity_challenge_chat/chat_scripts/pages/chat_script_page.dart

```diff
@@ -17,28 +21,34 @@ class ChatScriptPage extends StatelessWidget {
         }
       },
       builder: (BuildContext context, ChatScriptsState state) {
+        //do not show back for the first day chat after intro
+        final showBackButton = context
+                .read<GenerosityChallengeCubit>()
+                .state
+                .availableChatDayIndex !=
```

Comment:
So I can understand this PR better, "availableChatDayIndex" refers to the "latest" available day (so not the current active index or anything else)?

Reply:
availableChatDayIndex actually it refers to the first non completed chat. There could be cases when active day index is not the same as active day chat index (for ex user still needs to pass through post chat of day 1, but the day 2 already available)